### PR TITLE
W22 6pm 2 zl personal schedules table

### LINKED ISSUE #29: Implement Table component for PersonalSchedules
Implement Table component for PersonalSchedules

Create a table component for PersonalSchedules. See team03 assignment for examples.

# ACs

- [x] The table has columns for database fields (i.e. the attributes of the corresponding `@Entity` class
- [x] There are fixtures under `/frontend/src/fixtures` for this data type
- [x] There is a storybook entry
- [x] 100% coverage with `npm run coverage`
- [x] 100% coverage with `npx stryker run`

### PR DESCRIPTION
**Pull request by ZeyuanLu**
# Overview

Added table component for personal schedules
Added test for table component

# Issues Addressed

closes #29 
storybook
image

### CODE REVIEW

**Review by Lai-Justin — DISMISSED** [review]
LGTM

File: frontend/src/fixtures/personalSchedulesFixtures.js
Comment (pconrad) [inline frontend/src/fixtures/personalSchedulesFixtures.js:6]: Quarter should always be stored in YYYYQ format internally.

**Comment by pconrad** [comment]
This looks good... it would be better with this addition (but this can be another issue):
https://github.com/ucsb-cs156-w22/team04-w22-6pm-courses/issues/41

**Review by pconrad — CHANGES_REQUESTED** [review]
Please change the fixtures for the personalSchedules to YYYYQ format.

Not just on this line, but everywhere that `quarter` appears.

File: frontend/src/fixtures/personalSchedulesFixtures.js
Comment (alexactss) [reply frontend/src/fixtures/personalSchedulesFixtures.js]: Fixed.

**Review by alexactss — DISMISSED** [review]
LGTM

**Review by pconrad — APPROVED** [review]

**Review by pconrad — DISMISSED** [review]
LGTM... will test on Heroku before merging

**Comment by pconrad** [comment]
Deploying here: https://dashboard.heroku.com/apps/team04-w22-6pm-2-qa/deploy/github

File: frontend/src/main/utils/PersonalScheduleUtils.js
Comment (andrewhlu) [inline frontend/src/main/utils/PersonalScheduleUtils.js:16]: Do you foresee that you will be using these in more than one location? If not, I'm curious as to why you decided to factor these functions into a separate util file instead of keeping this in the component directly.

**Review by andrewhlu — CHANGES_REQUESTED** [review]
One question that I had about your code, see below.

Also, I noticed that the Storybook screenshot in your PR description is for the courses table, not the personal schedules table. Please amend the PR description to show the correct table. It would also help to paste a link to the deployed QA storybook for people to look at.
(Additionally, it isn't necessary to include statements / screenshots of 100% test coverage - that is implied / required with every pull request)

File: frontend/src/main/utils/PersonalScheduleUtils.js
Comment (alexactss) [reply frontend/src/main/utils/PersonalScheduleUtils.js]: Fixed

**Comment by pconrad** [comment]
Storybook is here: https://ucsb-cs156-w22.github.io/team04-w22-6pm-courses-docs-qa/storybook-qa/w22-6pm-2-zl-personalSchedulesTable/?path=/story/components-coursesearch-subjectselector--all-the-subjects

**Review by pconrad — APPROVED** [review]
LGTM--waiting on CI/CD then will merge

**Review by alexactss — APPROVED** [review]
LGTM

**Pull request merged**
